config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_secretmanagersecretversions.secretmanager.cnrm.cloud.google.com.yaml

```diff
@@ -135,25 +128,24 @@ spec:
                   - external
                 properties:
                   external:
-                    description: 'Allowed value: The `name` field of a `SecretManagerSecret`
-                      resource.'
+                    description: A reference to an externally managed SecretManagerSecret
+                      resource. Should be in the format "projects/<projectID>/locations/<location>/secrets/<secretID>".
                     type: string
                   name:
-                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
+                    description: The name of a SecretManagerSecret resource.
                     type: string
                   namespace:
-                    description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/'
+                    description: The namespace of a SecretManagerSecret resource.
                     type: string
                 type: object
-            required:
-            - secretData
```

Comment:
Probably worth adding the `+required` annotations just to avoid this change (though it is allowed by API evolution rules)